README.adoc

```diff
@@ -324,7 +324,7 @@ Standard Maven build:
 
 == Testing
 
-http://spockframework.org/(Spock) is used for testing the calling of the Mojo. This will be downloaded by Maven. Tests are run simply by:
+link:http://spockframework.org/[Spock] is used for testing the calling of the Mojo. This will be downloaded by Maven. Tests are run simply by:
```

Comment:
The `link:` part isn't necessary. Simply changing the round brackets to square brackets will do it.